kolibri/plugins/document_pdf_render/assets/src/views/index.vue

```diff
@@ -39,6 +62,9 @@
       mimicFullscreen() {
         return !this.fullscreenAllowed && this.isFullscreen;
       },
+      targetTime() {
+        return this.totalPages * 30;
```

Comment:
No - the only stats available from PDFJS are to do with fonts and streams - and any metadata that happens to be attached to the PDF by the author.